server/subscriptions.go

```diff
@@ -8,23 +8,22 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/google/go-github/v31/github"
 	"github.com/mattermost/mattermost-server/v5/mlog"
-
-	"github.com/google/go-github/v25/github"
 )
 
 const (
-	SUBSCRIPTIONS_KEY       = "subscriptions"
-	EXCLUDE_ORG_MEMBER_FLAG = "exclude-org-member"
+	SubscriptionsKey     = "subscriptions"
+	excludeOrgMemberFlag = "exclude-org-member"
 )
 
 type SubscriptionFlags struct {
 	ExcludeOrgMembers bool
 }
 
 func (s *SubscriptionFlags) AddFlag(flag string) {
-	switch flag {
-	case EXCLUDE_ORG_MEMBER_FLAG:
+	switch flag { // nolint:gocritic
```

Comment:
What's the error here that we are ignoring ?

Reply:
https://go-critic.github.io/overview#singleCaseSwitch-ref

Reply:
Right .. so any reason we can't convert it into a if condition ?

Reply:
I expect that more `flags`s will be added in the future and I want the code to be in a place where they can easily be added to the `switch'.

Reply:
Sounds good.